packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardBack.tsx

```diff
@@ -40,14 +40,15 @@ const JurisdictionCardBack: FC<TJurisdictionCardBackProps> = ({ setIsFlipped, ve
     if (verificationDocs)
         return (
             <div className='wallets-jurisdiction-card-back'>
-                <LegacyArrowLeft2pxIcon
+                <button
```

Comment:
Can we use IconButton component instead?

Reply:
It has different properties (full width, extra padding around the icon and hover effect) which are unneeded